packages/story-editor/src/masks/index.js

```diff
@@ -32,6 +32,11 @@ export function canMaskHaveBorder(element) {
   return !mask || mask.supportsBorder;
 }
 
+export function checkMultiBorderSupport(element) {
```

Comment:
Nit: Would be nice to have this start with `can...` as well, e.g. `canHaveMultiBorder`. `check...` doesn't really say it will return a boolean, starting with `has...` or `can...` would be great.

Reply:
Renamed to `canSupportMultiBorder()` in 9165f3f9e.